ansible/roles/prometheus/tasks/config.yml

```diff
@@ -191,7 +190,7 @@
     service: "{{ prometheus_services['prometheus-snmp-exporter'] }}"
   command: >
     docker run --rm \
-      --mount type=bind,src={{ node_config_directory }}/prometheus-snmp-exporter,dst=/etc/prometheus \
+      --mount type=bind,src={{ node_config_directory }}/prometheus-snmp-exporter,dst=/opt \
```

Comment:
What is the reason for this? IPMI exporter looks here now?

Reply:
I need to revisit this post xena-upgrade, it's aged a bit. Moving it back to draft for now.